### DISCUSSION TITLE
Any concerns with updating Google Authentication

### DISCUSSION THREAD

**Discussion by budhoward** [dnnsoftware/Dnn.Platform#4640] [General]
I have been using Google Authentication for a long while on some of my sites.  After recently upgrading to DNN 9.3.2 I see there is a 9.3.2 version of "DotNetNuke Google Authentication Project".  My sites are currently using 9.1.1.  Is there any (known) risk to installing the 9.3.2 version?

**Comment by valadas** [comment]
I don't think it would be a problem if it does install, that being said:

1. It's not new to 9.3.2, so you could also grab it from the 9.1.1 install packages to be sure.
2. I would strongly advice upgrading DNN instead 😄

**Reply by budhoward** [reply]
I think I confused the issue.  My DNN site version is 9.3.2, however, the Google authentication is listed as 9.1.1.  I expected that when I upgraded to 9.3.2 it would have upgraded the Google authentication to 9.3.2.

So, can I do that manually without breaking the authentication of users currently in place?

**Reply by valadas** [reply]
Oh, the build scripts before did not make all modules follow the release version, nowadays all extensions distributed with the main distribution follow the distribution version number. So back then you may have had 9.1.1 delivered with DNN 9.3.2 but right now it would match the DNN release version it came from. That being said, I am not aware of any remotely recent changes to it, so it probably is exactly the same for all these versions 😄

**Reply by budhoward** [reply]
Thank you.  I was successfully able to install the 9.3.2 version, validation was successful, but I'll hang on to backups of app & db until I'm comfortable I won't need to back out.

Thank you
